Dockerfile-ubuntu.template

```diff
@@ -145,6 +149,8 @@ RUN $OPENSSL_INSTALL_PATH_PREFIX/bin/openssl version
 
 FROM openssl-builder as erlang-builder
 
+ARG BUILDKIT_SBOM_SCAN_STAGE=true
```

Comment:
Only the `erlang-builder` stage needs its SBOM captured since it is from the previous stage, `openssl-builder`, which is from the previous one, `build-base`, so having all three would just add duplication in the final SBOM.

Reply:
I agree that the base doesn't need them but the end image pull files from both the `openssl-builder` and `erlang-builder` image so I think they both need to have it